pkg/graph/schema.graphql

```diff
@@ -1332,6 +1343,27 @@ input PlanCrTdlChanges @goModel(model: "map[string]interface{}") {
     note: String
 }
 
+
+type AuditChange {
+  id: Int!
+  primaryKey: UUID!
+  foreignKey: UUID
+  tableName: String!
+  action: String!
+  fields: Map!
+  modifiedBy: String
+  modifiedDts: Time
+}
+
+# type AuditFields {
+#   AuditField[]
+
+# }
+
+# type AuditField {
+#   old: String
+#   new: String
+# }
```

Comment:
Can these lines be removed? 
```suggestion
```

Reply:
Thanks @rileyanderson , I removed it. I had left it because I thought we'd likely change to this in the future, but it is simple enough that we can recreate it later.